tt_llk_blackhole/llk_lib/llk_math_eltwise_binary_sfpu_params.h

```diff
@@ -0,0 +1,62 @@
+// SPDX-FileCopyrightText: © 2025 Tenstorrent AI ULC
+//
+// SPDX-License-Identifier: Apache-2.0
+
+#pragma once
+#include "llk_math_eltwise_binary_sfpu.h"
+#include "llk_sfpu_types.h"
+
+template <bool APPROXIMATE, class F, class... ARGS>
+inline void _llk_math_eltwise_binary_sfpu_params_(F&& sfpu_func, uint dst_index0, uint dst_index1, int vector_mode = (int)VectorMode::RC, ARGS&&... args)
+{
+    uint dst_index  = (dst_index0 <= dst_index1) ? dst_index0 : dst_index1;
+    uint dst_offset = (dst_index0 > dst_index1) ? dst_index0 - dst_index1 : dst_index1 - dst_index0;
+    _llk_math_eltwise_binary_sfpu_start_<DST_SYNC_MODE>(dst_index);
+
+    if (vector_mode == (int)VectorMode::R)
+    {
+        // Do a row vector, Face0 + Face1 -- first iteration (first row)
+        const int ITERATIONS = 1;
+#pragma GCC unroll 0
+        for (int face = 0; face < 2; face++)
+        {
+            sfpu_func(dst_offset, static_cast<ARGS&&>(args)...);
```

Comment:
```suggestion
            std::forward<F>(sfpu_func)(dst_offset, std::forward<ARGS>(args)...);
```
Don't forget to `#include <utility>` if you decide to follow this suggestion through.